frontends/web/src/api/testing.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2022 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { apiGet } from '../utils/request';
+
+export const getTesting = (): Promise<boolean> => {
```

Comment:
imo one backend.ts file is better than one file per api call (version.ts, testing.ts, etc.) A new file imho makes sense for sub-routes, e.g. `/aopp/...` or the accounts api.

Reply:
I agree having `api/using-mobile-data.ts` and `api/testing.ts` is ugly, but

IMO `backend.ts` has no meaning and shouldn't exist as everything is backend api and I'd rather refactor/rename the endpoints https://github.com/digitalbitbox/bitbox-wallet-app/blob/55d685c797cb2db5695e944c57ce3a8bce26c250/backend/handlers/handlers.go#L167-L206